### ISSUE TITLE
0: Uncaught error

### ISSUE THREAD

**Issue by legacy6vz** [JakeMakesStuff/MagicCap#237]
## Information
`MagicCap Version: 2.1.2
System OS: Darwin 20.5.0 / Platform: darwin
Installation ID: 070f000b-b0aa-4f5c-9fa9-db51569a5d45
Config: {"clipboard_action":2,"clipboard_hotkey":null,"custom_body":{},"custom_headers":{"Authorization":"[REDACTED]","domain":"cdn.6vz.pw","service":"magiccap"},"custom_image":"file","custom_response":"url","custom_url":"https://api.tixte.com/v1/upload","gif_hotkey":null,"hotkey":"Cmd+num1","install_id":"070f000b-b0aa-4f5c-9fa9-db51569a5d45","light_theme":true,"open_login":true,"save_capture":true,"save_path":"/Users/[REDACTED]/Pictures/MagicCap/","ultra_https":false,"upload_capture":true,"uploader_type":"custom"}`

### Description 
I just wanted to do a screenshot, but it refuses to do it, because of "0: Uncaught Error"

### Steps to reproduce
Do a screenshot, or smh

### Expected behavior
This error should appear

### Screenshots
https://imgur.com/aLtiuuZ

**Comment by mrhappyma** [comment]
Same thing is happening to me

**Comment by milanmdev** [comment]
I'm having the same issue - but with the current state of the project, I don't see this getting fixed any time soon.

**Comment by red-socks294** [comment]
Same

Error: Upload fetch error: Error: 0: Uncaught Error. You should report this to the GitHub!
is what I got, i tested

**Comment by milanmdev** [comment]
For anyone here using Tixte as their image upload service, they recently released their own macOS screenshot client (soon to be available to Linux).

To download it, login to your account and head over to [this page](https://tixte.com/dashboard/integrations) (https://tixte.com/dashboard/integrations).

**Comment by red-socks294** [comment]
I'm using [e-z.host](https://e-z.host) as my upload service.

**Comment by mrhappyma** [comment]
> For anyone here using Tixte as their image upload service, they recently released their own macOS screenshot client (soon to be available to Linux).
> 
> To download it, login to your account and head over to [this page](https://tixte.com/dashboard/integrations) (https://tixte.com/dashboard/integrations).

yes, and it is very nice but is very limited on customization/configurability. we need something as versatile as sharex, but it seems that will forever only be for windows users.
